Core/Utilities/Graphs/GraphData.py

```diff
@@ -401,7 +401,7 @@ def initialize( self ):
       self.keys = self.sortKeys()
 
     self.values = [ self.parsed_data[k] for k in self.keys ]
```

Comment:
Could you please also add a protection to this line as well?
self.values = [ self.parsed_data[k] if k in self.parsed_data else for k in self.keys ]